docs/view_components/flow_chart.rst

```diff
@@ -1,11 +1,57 @@
-Component title
----------------
+.. _flowchart-label:
+
+Flow-chart
+----------
+The flow chart is a view component that shows all the sequential steps of the tool in order to get results.
+
+A flow chart view-component has a number of advantages:
+
+* Helps in sequentially progressing through the modeling and simulation process
+* Allows the user to jump between different steps of the simulation
+* Brings a design consistency and uniformity to the UI of the tool
+* Splits the tool in different views which increases the user friendliness 
 
 Attributes
 ^^^^^^^^^^
 
+* Each button is a step in the simulation process
+* The list of steps in the progression bar are as follows:
+    #. Project Setup
+    #. Inputs
+    #. System Inputs
+    #. Visualization
+    #. Constraints
+    #. Simulations
+    #. Results
+* Each step has an associated index, with the progression being bar considered as an array
+* The current state of the view depends on the index of the current step
```

Comment:
This is more an action/requirement

Reply:
I would keep it as an attribute because the view could have been the default where there was no requirement for the user to have done something to reach that state.